builtin/logical/pki/path_tidy_test.go

```diff
@@ -548,6 +554,7 @@ func TestTidyIssuerConfig(t *testing.T) {
 	defaultConfigMap["interval_duration"] = int(time.Duration(defaultConfigMap["interval_duration"].(float64)) / time.Second)
 	defaultConfigMap["issuer_safety_buffer"] = int(time.Duration(defaultConfigMap["issuer_safety_buffer"].(float64)) / time.Second)
 	defaultConfigMap["safety_buffer"] = int(time.Duration(defaultConfigMap["safety_buffer"].(float64)) / time.Second)
+	defaultConfigMap["revoked_safety_buffer"] = int(time.Duration(defaultConfigMap["revoked_safety_buffer"].(float64)) / time.Second)
```

Comment:
In some of these places, while the response will include `revoked_safety_buffer` (because we always want to reply with it, even if it is `nil`), we probably will refer to the value of `safety_buffer` in those cases.